content/en/docs/zero-code/java/spring-boot-starter/additional-instrumentations.md

```diff
@@ -36,7 +36,7 @@ instrumentation library.
 | --------------------------------------------- | ------- | ------- | ----------------------------------------------------------------------------------------------- |
 | `otel.instrumentation.log4j-appender.enabled` | Boolean | true    | Enables the configuration of the Log4j OpenTelemetry appender with an `OpenTelemetry` instance. |
 
-## OpenTelemetry instrumentations libraries
+## Instrumentations libraries
```

Comment:
```suggestion
## Instrumentation libraries
```

Reply:
Oh right, thanks for catching that @cartermp.